src/js/storyTelling-page-storyEdit.js

```diff
@@ -93,8 +95,77 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
                 type: "sjrk.storyTelling.ui.storyViewer",
                 container: ".sjrkc-st-story-previewer",
                 options: {
+                    selectors: {
+                        progressArea: ".sjrkc-st-story-share-progress",
+                        responseArea: ".sjrkc-st-story-share-response",
+                        responseText: ".sjrkc-st-story-share-response-text"
+                    },
                     listeners: {
-                        "onStoryViewerPreviousRequested.requestContextChange": "{page}.events.onContextChangeRequested.fire"
+                        "onStoryViewerPreviousRequested.requestContextChange": "{page}.events.onContextChangeRequested.fire",
+                        "onShareRequested": [{
+                            func: "{that}.showProgressArea",
+                            namespace: "showProgressArea"
+                        },{
+                            func: "{that}.disableShareButton",
+                            namespace: "disableShareButton"
+                        },{
+                            func: "{that}.hideServerResponse",
+                            namespace: "hideServerResponse"
+                        }],
+                        "onShareComplete": [{
+                            func: "{that}.hideProgressArea",
+                            namespace: "hideProgressArea"
+                        },{
+                            func: "{that}.enableShareButton",
+                            namespace: "enableShareButton"
+                        },{
+                            func: "{that}.setServerResponse",
+                            args: ["{arguments}.0"],
+                            namespace: "setServerResponse"
+                        },{
+                            func: "{that}.showServerResponse",
+                            namespace: "showServerResponse"
+                        }]
+                    },
+                    invokers: {
+                        setShareButtonDisabled: {
+                            this: "{that}.dom.storyShare",
+                            method: "prop",
+                            args: ["disabled", "{arguments}.0"]
+                        },
+                        enableShareButton: {
+                            func: "{that}.setShareButtonDisabled",
+                            args: [false]
+                        },
+                        disableShareButton: {
+                            func: "{that}.setShareButtonDisabled",
+                            args: [true]
+                        },
+                        showProgressArea: {
+                            this: "{that}.dom.progressArea",
+                            method: "show",
+                            args: [0]
```

Comment:
Why `args: [0]`? The same comment applies to other invokers below.

Reply:
According to the docs for the jQuery `show` and `hide` methods, the default is to apply an animation to the operation. I don't want the animation to show, so I'm passing in a value of `0` to the first argument, `duration`.

documentation here: http://api.jquery.com/show/

Reply:
I see. Thanks for the explanation.